posthog/models/filters/stickiness_filter.py

```diff
@@ -36,11 +37,11 @@ def __init__(self, data: Optional[Dict[str, Any]] = None, request: Optional[Http
         super().__init__(data, request, **kwargs)
         team: Optional[Team] = kwargs.get("team", None)
         if not team:
-            raise ValueError("Team must be provided to stickiness filter")
+            raise ValidationError("Team must be provided to stickiness filter")
         self.team = team
         get_earliest_timestamp: Optional[Callable] = kwargs.get("get_earliest_timestamp", None)
         if not get_earliest_timestamp:
-            raise ValueError("Callable must be provided when date filtering is all time")
+            raise ValidationError("Callable must be provided when date filtering is all time")
```

Comment:
Hmm maybe this should be a `ValueError`? Doesn't seem like this is something the user can fix, if it's not set it's probably bad code.

Reply:
It's someone trying to reverse-engineer the API and not passing in the right arguments for the endpoint -> it falls under this umbrella.